variants/frontend-react/jest.config.js

```diff
@@ -0,0 +1,12 @@
+const config = {
+  testEnvironment: 'node',
+  clearMocks: true,
+  restoreMocks: true,
+  resetMocks: true,
+
+  moduleDirectories: ['node_modules', 'app/frontend'],
+
+  setupFilesAfterEnv: ['app/frontend/tests/setupExpectEachTestHasAssertions.js']
```

Comment:
```suggestion
  setupFilesAfterEnv: [
    '@testing-library/jest-dom/extend-expect',
    'app/frontend/tests/setupExpectEachTestHasAssertions.js'
  ]
```

This'll mean we don't have to have this import in every test.

(Technically we don't have to have the `/extend-expect` at the end, but think it makes it more obvious what this is doing)